packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -510,3 +510,13 @@ export const getSignature = (name: string) => {
 export const getSignatureWithNote = (name: string, note: string) => {
   return `${getSignature(name)}: ${note}\n`;
 };
+
+export const userCanVote = (user: UsersMinimumInfo|DbUser) => {
+  if (!isLW) return true;
+
+  // If the user doesn't have a `createdAt`, the date comparison will return false, which then requires them passing the karma check
+  const userCreatedAfterCutoff = new Date(user.createdAt) > new Date(lowKarmaUserVotingCutoffDateSetting.get());
+  const userKarmaAboveThreshold = user.karma > lowKarmaUserVotingCutoffKarmaSetting.get();
+
+  return !userCreatedAfterCutoff || userKarmaAboveThreshold;
+};
```

Comment:
I think this is very similar to a function we already have, although the existing function takes in the thing being voted on. Maybe that's fine.

Reply:
@jpaddison3 do you remember what function?  We have a lot of different things related to voting permissions

Reply:
[Useful recent PR](https://github.com/ForumMagnum/ForumMagnum/pull/6739/files) to look at. It looks like [makeVotable](https://github.com/ForumMagnum/ForumMagnum/pull/6739/files#diff-9cdfccbbde9db0d3f516c7027d829e305d0da83ec6ca82706756e5d147e9c51aR113) has something interesting.

Reply:
It looks like `userCanVoteOn` doesn't have a default, and I'd need to write some additional logic to:
1. provide a default
2. "merge" it with the per-collection logic

I'm sort of wary of that kind of merging; from experience that kind of default behavior causes headaches when someone else is trying to track down some unexpected behavior later.  I don't love the current situation with permissions but that doesn't feel like an obvious improvement.